Assets/MixedRealityToolkit-Examples/Demos/Boundary/Prefabs.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
This meta doesn't have a corresponding folder.

![image](https://user-images.githubusercontent.com/3580640/42978774-9248cf02-8b83-11e8-84af-3966ed2fa705.png)

Reply:
missed deleting this after I moved files around..